third_party/aws-sdk-auth-lite/README.md

```diff
@@ -0,0 +1,8 @@
+# aws-sdk-auth-lite
```

Comment:
I am worried about the maintainability. How would e.g. I go about updating this code to the latest SDK version?

I think it would be great if you could document the steps you did to arrive at aws-sdk-auth-lite and check in the code modifications you did as a patch, so that we can version it.

Reply:
I ripped out anything that wasnt needed to deal with anything outside of loading credentials. I dont know how to make it anything other than manual.

I am not 100% sure if a patch is going to be great for this, but I can generate one.

Does bazel have rules for applying a patch over sources?

Reply:
Ok errrr, might have to find a slightly better way to arrange this tree.

A patch generated weighs in at 2.9Gb